integration-test/src/test/java/io/lumify/it/VertexIntegrationTest.java

```diff
@@ -49,7 +49,7 @@ public void testSearchVisibleWithQueryString() throws ApiException {
         List<ClientApiVertex> vertices;
 
         // matches all visible
-        vertices = helper.vertexApi.vertexSearch(PROPERTY_QUERY_STRING, EMPTY_FILTER, null, null, null, null,
+        vertices = helper.vertexApi.vertexSearch(PROPERTY_QUERY_STRING + "*", EMPTY_FILTER, null, null, null, null,
```

Comment:
Just want to make sure this change belongs in the test. Should the vertexSearch method be auto-wildcarding or is it supposed to execute exactly what it's given? What do the unit tests for that method indicate?